tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py

```diff
@@ -16,6 +16,7 @@
     skipif_ocp_version,
     performance,
     performance_b,
+    ignore_leftovers,
```

Comment:
In-case of failures Ignore leftovers will leave the cluster space utilised since it's clone test.
Would be good to handle it to delete the clones and to release the data.

Reply:
Sure, we delete the clones in the teardown (lines 116-122)